bot/exts/help_channels/_cog.py

```diff
@@ -20,6 +21,9 @@
 HELP_CHANNEL_TOPIC = """
 This is a Python help channel. You can claim your own help channel in the Python Help: Available category.
 """
+AVAILABLE_HELP_CHANNELS = """
+Currently available help channel(s): {available}
```

Comment:
How does it look if we do this:
```suggestion
**Currently available help channel(s)**: {available}
```

Reply:
![image](https://user-images.githubusercontent.com/15021300/106986694-bea48700-6720-11eb-868b-17b1b0e29dad.png)

I prefer the first one.

Reply:
Hmm, i like it with bold, let's see what others prefer.

Reply:
I think it looks fine normally (without bold).

Reply:
I prefer the bold version to catch the eyes

Reply:
After further consideration I see the appeal of the bold. I will change this now.